README.md

```diff
@@ -824,6 +825,228 @@ and Equinox will supply the _initial_ value for the `project` function to render
+
+#### The ingredients
+
+With Deciders in general, and Equinox in particular, the following elements are involved:
+- a `State` type, on which we base decisions, which can be updated by the consequences of a decision
+- a `decide` function, which is presented a `State`, and returns a decision, which we use to update the State, if, and only if there is no concurrency conflict when applying them
+- an `initial` State, from which we start
+- an `Event` type (think a Discriminated union)
+- the Events are maintained as an ordered list of events (could be an in memory array, a stream in ESDB, documents with a Tip as with `Equinox.CosmosStore`)
+- the State is established by [`fold`](https://en.wikipedia.org/wiki/Fold_(higher-order_function))ing the sequence of `Event`s, starting from an `initial` state
+
+With the Equinox `type Decider`, the typical `decide` signature used with the `Transact` API has the following signature:
+
+      context -> inputsAndOrCommand -> 'State -> Event list
+
+> NOTE: There are more advanced forms that allow the `decide` function to be `Async` and/or to also return a `'result`, which will be yielded to the caller driving the Decision as the return value of the `Transact` function.
+
+#### So what kind of a thing is a Decider then?
+
+Is it a marketing term? Jérémie's way of explaining an app?
+
+I'd present the fact that Equinox:
+- was initially generalised and extracted from working code using ESDB in (by most measures) a successful startup written in a mix of F# and C# by the usual mix of devs, experience levels, awareness of event sourcing patterns and language/domain backgrounds
+- for a long time only had its MemoryStore as the thing to force it to be store agnostic
+- did not fundamentally change to add idiomatic support for a Document database (CosmosDB)
+- will not fundamentally change to add idiomatic support for DynamoDB
+- can and has been used at every stage in an e-commerce pipeline
+- is presently aligning pretty neatly with diverse domains without any changes/extensions, both for me and others
+
+... as evidence for Decider being *a pattern language* (analogous to how event sourcing and many modern event sourced UI paradigms share a lot of common patterns).
+
+#### ... about the process of making _decisions_
+
+Finally, I'd say that a key thing the Decider concept brings is a richer way of looking at event sourcing than the typical event sourcing 101 examples you might see:
+- de-emphasizing one-way things that map commands to events without deduplicating and/or yielding a result (not saying that you shouldn't do the simplest thing -- you absolutely should)
+- de-emphasizing all projection handlers only ever just sitting there looking for `MyThingCreated` and doing an `INSERT` with a try/catch for duplicate inserts as a way to make it idempotent (and every stream design requiring a Created event as a way to enable that)
```

Comment:
**SMALL SUGGESTION:** I think breaking a long sentence like that into shorter sentences would help the newbie get the point.

Reply:
Would appreciate a suggestion as I have no idea what to say! You can supply a suggested edit to go with the PR when I post it ;)